hydromt_wflow/naming.py

```diff
@@ -359,18 +359,18 @@
     "RiverSlope": {
         "wflow_v0": "lateral.river.slope",
         "wflow_v1": "river__slope",
-        "hydromt_name": "rivslp",
+        "hydromt_name": "river_slope",
     },
     "wflow_riverwidth": {
         "wflow_v0": "lateral.river.width",
         "wflow_v1": "river__width",
-        "hydromt_name": "rivwth",
+        "hydromt_name": "river_width",
     },
     # lakes
     "LakeArea": {
         "wflow_v0": "lateral.river.lake.area",
         "wflow_v1": "lake_surface__area",
-        "hydromt_name": "LakeArea",
+        "hydromt_name": "lake_area",
```

Comment:
For lakes and reservoirs I also want to change the hydromt name as well as the default staticmaps name (the hydromt name is used in update mode, if you already have lakes.geojson in your geoms and use it to update some of the parameters of your lakes rather than go back and update the globla data).